backend/plugins/authorization.js

```diff
@@ -12,7 +13,9 @@ const ESCAPED_ROUTES = [
 ]
 
 function accessTokenIsValid(access_token) {
-    return getSetting('ACCESS_TOKEN') === access_token;
+    const currentTime = Date.now();
+    return (getSetting('ACCESS_TOKEN') === access_token &&
+            getSetting('ACCESS_TOKEN_EXPIRY') > currentTime);
```

Comment:
We might need to `parseInt` this setting. I'm not sure.

Reply:
if it's coming from the `env`, then we `JSON.parse` it which will turn it into a number: https://github.com/plotly/plotly-database-connector/blob/master/backend/settings.js#L160

```
> typeof JSON.parse('4')
'number'
```

but i'm not sure what the type will be if it comes from YAML

Reply:
it's probably worth throwing in a quick test in https://github.com/plotly/plotly-database-connector/blob/master/test/backend/settings.spec.js for it:

```
it('Parses number settings as numbers', ()  => {
    const now = Date.now();
    saveSetting('ACCESS_TOKEN_EXPIRY', now);
    const nowSetting = getSetting('ACCESS_TOKEN_EXPIRY');
    expect(typeof nowSetting, 'number');
    expect(nowSetting, now);
    
    const anotherNow = now + 1;
    process.env.PLOTLY_CONNECTOR_ACCESS_TOKEN_EXPIRY = String(anotherNow);
    const anotherNowSetting = getSetting('ACCESS_TOKEN_EXPIRY');
    expect(typeof anotherNowSetting, 'number');
    expect(anotherNowSetting, anotherNow);
});
```

That way we can rest easy if ever update our YAML dep or otherwise modify `getSetting`

Reply:
Checked, and it seems it is `number` so we should be good w/o `parseInt`. Moreover if this behavior changed in future, ⬆️ these tests (added!) will let us know.. :)